legacy/utils.js

```diff
@@ -545,3 +546,28 @@ async function getLowestVariantValuesList(valuesList, namespace) {
   // return each product's list of SKUs
   return Array.from(productSKUListMap.values()).flat();
 }
+
+// converts attribute value's value__c's SObject ID to Product ID for Product Reference fields
+async function parseProductReferenceAttrVal(attrValValue, reqBody) {
+  if (!attrValValue) return '';
+  const service = new ForceService(reqBody.hostUrl, reqBody.sessionId);
+  const helper = new PimExportHelper(reqBody.namespace);
+
+  // split the value to get the individual Product__c SObject Ids
+  let productSobjectIds = attrValValue.split(', ');
+
+  productSobjectIds = await module.exports.prepareIdsForSOQL(productSobjectIds);
+  let products = await service.simpleQuery(
+    helper.namespaceQuery(
+      `select Id, Name
+      from Product__c 
+      where Id IN (${productSobjectIds})`
+    )
+  );
+  products = products
```

Comment:
optional: no  need for reassignment can just return the entire expression 

we shld start documenting what the return types are of methods